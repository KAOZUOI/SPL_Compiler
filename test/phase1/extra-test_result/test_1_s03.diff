1,4d0
< Error type A at Line 1: unknown lexeme 
< Error type B at Line 1: Missing semicolon ';'
< Error type A at Line 2: unknown lexeme 
< Error type A at Line 3: unknown lexeme 
6d1
< Error type A at Line 4: unknown lexeme 
8,23c3,7
< Error type A at Line 5: unknown lexeme 
< Error type A at Line 6: unknown lexeme 
< Error type A at Line 7: unknown lexeme 
< Error type A at Line 8: unknown lexeme 
< Error type A at Line 9: unknown lexeme 
< Error type A at Line 10: unknown lexeme 
< Error type A at Line 11: unknown lexeme 
< Error type A at Line 12: unknown lexeme 
< Error type A at Line 13: unknown lexeme 
< Error type A at Line 14: unknown lexeme 
< Error type A at Line 15: unknown lexeme 
< Error type A at Line 16: unknown lexeme 
< Error type A at Line 17: unknown lexeme 
< Error type A at Line 18: unknown lexeme 
< Error type A at Line 19: unknown lexeme 
< Error type A at Line 20: unknown lexeme 
---
> Error type A at Line 7: unknown lexeme 0x023456
> Error type A at Line 9: unknown lexeme 0023
> Error type A at Line 11: unknown lexeme 000.234
> Error type B at Line 12: Missing closing parenthesis ')'
> Error type B at Line 16: Missing closing parenthesis ')'
