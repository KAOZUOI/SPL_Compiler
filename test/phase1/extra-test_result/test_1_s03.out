Error type A at Line 1: unknown lexeme 
Error type B at Line 1: Missing semicolon ';' LOC2
Error type A at Line 2: unknown lexeme 
Error type A at Line 3: unknown lexeme 
Error type A at Line 4: unknown lexeme '\xr0'
Error type A at Line 4: unknown lexeme 
Error type A at Line 5: unknown lexeme 2_A
Error type A at Line 5: unknown lexeme 
HEX_INT: 255
Error type A at Line 6: unknown lexeme 
HEX_INT: 144470
Error type A at Line 7: unknown lexeme 
HEX_INT: 0
Error type A at Line 8: unknown lexeme 
Error type A at Line 9: unknown lexeme 
Error type A at Line 10: unknown lexeme 
Error type A at Line 11: unknown lexeme 
Error type A at Line 12: unknown lexeme 
Error type A at Line 13: unknown lexeme 
Error type A at Line 14: unknown lexeme 
Error type A at Line 15: unknown lexeme 
Error type A at Line 16: unknown lexeme 
Error type A at Line 17: unknown lexeme 
Error type A at Line 18: unknown lexeme 
Error type A at Line 19: unknown lexeme 
Error type A at Line 20: unknown lexeme 
